pkg/plugins/policies/meshtls/api/v1alpha1/testdata/invalid-top-level-sectionName.output.yaml

```diff
@@ -1,3 +1,3 @@
 violations:
-  - field: spec.targetRef.sectionName
-    message: can only be used with inbound policies
\ No newline at end of file
+- field: spec.targetRef.sectionName
```

Comment:
you should probably commit this as a separate PR

Reply:
I was thinking about it, but I decided to keep it here is it is related to inbounds. I you want, I can move it to separate pr